packages/ui/src/components/Table/styles.module.css

```diff
@@ -10,4 +10,11 @@
 
 .table__searchInputContainer > div {
     width: 100%;
-}
\ No newline at end of file
+}
+
+.loaderContainer {
```

Comment:
this won't be exported in the lib, they only work on stories. If you want to style the spinner just use `sx` or other methods of styling from Material UI